fic_modules/git.py

```diff
@@ -167,7 +167,7 @@ def filter_git_tag_bp(repository_name, repository_path):
 
     if limit_checker():
         new_commits = GIT.get_repo(repository_path) \
-                         .get_commits(since=last_checked)
+            .get_commits(since=last_checked)
```

Comment:
.get() is weirdly indented.

Reply:
I had a look over this issue in Pycharm. It seems like trying to get `.get_commits(since=last_checked)` under `GIT.get_repo(repository_path) \` bring a PEP8 complain. It looks like, this form is correct according to PEP8. Same goes for the other weirdly indents.